.cbmc-batch/jobs/aws_cryptosdk_enc_ctx_serialize/Makefile

```diff
@@ -59,6 +59,11 @@ DEPENDENCIES += $(SRCDIR)/c-enc-sdk-src/enc_ctx.c
 
 ENTRY = aws_cryptosdk_enc_ctx_serialize_harness
 
+#Remove function bodies to deal with CBMC function pointer removal
+REMOVE_FUNCTION_BODIES += --remove-function-body aws_array_list_sort
+REMOVE_FUNCTION_BODIES += --remove-function-body aws_byte_buf_write
+REMOVE_FUNCTION_BODIES += --remove-function-body aws_cryptosdk_enc_ctx_size
+REMOVE_FUNCTION_BODIES += --remove-function-body aws_cryptosdk_hash_elems_array_init
```

Comment:
How are we sure this is safe?

Reply:
I didn't actually add the removal of these functions to the Makefile - they were in the original proof. I just changed ADDITIONAL_REMOVE_FUNCTION_BODY to REMOVE_FUNCTION_BODIES to better match the style I observed in other proofs. I can look further into ensuring its safe to remove them if needed.

Reply:
They're explicitly subbed above. So this is probably OK